app/views/users/new.html.erb

```diff
@@ -16,7 +16,7 @@
     <%= f.text_field :city %>
 
     <%= f.label :state %>
-    <%= f.text_field :state%><br>
+    <%= render partial: "shared/state_dropdown" %><br />
```

Comment:
@JeanJoeris is the Oprah of partials. 

"You get a partial, you get a partial. EVERYONE GETS A PARTIAL!"